src/couch/src/couch_httpd_auth.erl

```diff
@@ -227,22 +227,27 @@ jwt_authentication_handler(Req) ->
             RequiredClaims = get_configured_claims(),
             case jwtf:decode(?l2b(Jwt), [alg | RequiredClaims], fun jwtf_keystore:get/2) of
                 {ok, {Claims}} ->
+                    couch_log:info("Claims: ~p", [Claims]),
+                    Roles_Claim_Name = config:get(
+                        "jwt_auth", "roles_claim_name", "_couchdb.roles"
+                    ),
+                    Roles_Claim_Path = [list_to_binary(Item) || Item <- string:split(config:get(
+                        "jwt_auth", "roles_claim_path", ""
```

Comment:
I think it would be cleaner if roles_claim_path was independent of roles_claim_name. so you'd set it to "foo.bar.baz.roles". Ignore roles_claim_name if roles_claim_path is defined.

And obviously remove the couch_log:info lines.

Reply:
and use the shorter` ?l2b` and `?b2l` macros

Reply:
Should we mark `roles_claim_name` as deprecated (log a message, documentation) to only have one setting for this in future versions? I would suggest current behavior:

1. If `roles_claim_name` is found, use this - old behavior and doesn't break anything and log a info message to migrate to `roles_claim_path`
2. If `roles_claim_path` is found
  2.1 Use this
  2.2 If `roles_claim_name`is defined too, log an info message, that this value is ignoried, because of `roles_claim_path`